builtin/logical/aws/secret_access_keys.go

```diff
@@ -3,6 +3,9 @@ package aws
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/vault/sdk/framework"
```

Comment:
This `goimports` rearrangement looks wrong

Reply:
Resolved